inc/cli/class-migrate-command.php

```diff
@@ -0,0 +1,278 @@
+<?php
+/**
+ * Authorship.
+ *
+ * @package authorship
+ */
+
+declare( strict_types=1 );
+
+namespace Authorship\CLI;
+
+use WP_CLI;
+use WP_CLI_Command;
+use WP_Object_Cache;
+use WP_Post;
+use WP_Term;
+use WP_User;
+
+use const Authorship\GUEST_ROLE;
+
+/**
+ * Authorship migration commands.
+ */
+class Migrate_Command extends WP_CLI_Command {
+
+	/**
+	 * Migrates PublishPress Authors data to Authorship.
+	 *
+	 * Running this creates new Authorship data, but does not remove PPA data
+	 * or perform cleanup. If the new Authorship data is not good, delete it
+	 * and re-run as PPA's data will still be there.
+	 *
+	 * Guest authors that aren't mapped to a user will have a user created
+	 * with the guest author role and the same name.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--dry-run=<dry-run>]
+	 * : Perform a test run if true, make changes to the database if false.
+	 * ---
+	 * default: true
+	 * options:
+	 *   - true
+	 *   - false
+	 * ---
+	 *
+	 * [--overwrite-authors=<overwrite-authors>]
+	 * : If true overwrite Authorship data with publishpress data, if false
+	 * skip posts that already have Authorship data.
+	 * ---
+	 * default: false
+	 * options:
+	 *   - true
+	 *   - false
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp authorship migrate ppa --dry-run=true
+	 *
+	 * @when after_wp_load
+	 *
+	 * @param array<string> $args CLI arguments
+	 * @param array<string> $assoc_args CLI arguments
+	 */
+	function ppa( $args, $assoc_args ) : void {
+		if ( ! is_plugin_active( 'publishpress-authors/publishpress-authors.php' ) ) {
```

Comment:
```suggestion
		if ( ! taxonomy_exists( 'author' ) ) {
```

This is a bit more specific and means it'll still work if for example PPA is installed as a mu-plugin.